### PR TITLE
Added function to return nodes info

### PR DESCRIPTION
**Pull request by nishith-vihar**
The function return a dictionary with nodes info consisting
of hostname,ip,user and passwd.

Fixes: #118
Signed-off-by: Nishith Vihar Sakinala <nsakinal@redhat.com>

### CODE REVIEW

File: parsing/redant_params_handler.py [hunk 1]
```diff
@@ -76,6 +76,76 @@ def get_client_ip_list(cls) -> list:
...
+                            passwd : passwd
+                        },
+                        {
+                            hostname : hostname
+                            ip : ip
+                            user : user
+                            passwd : passwd
+                        }
+           ]
+        }
```
Comment (nik-redhat) [inline parsing/redant_params_handler.py:115]: nit: Can rephrase it to example format and instead provide some dummy values to make it more clear.
Like,
```
{
                            hostname : node1
                            ip : 10.70.1.121
                            user : root
                            passwd : password
                        },
```

File: parsing/redant_params_handler.py [hunk 2]
```diff
@@ -76,6 +76,76 @@ def get_client_ip_list(cls) -> list:
...
+    def get_nodes_info(cls) -> dict:
+        """
+        Returns a dictionary consisting of server info
+        Returns:
+            dict: dictionary consisting of server info
+        format of dictionary:
+        {
+            servers: [
+                        {
+                            hostname : hostname
```
Comment (srijan-sivakumar) [inline parsing/redant_params_handler.py:89]: "hostname" . These are json values. Keep the keys are strings. So it'd be better to reflect the same in example.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]
nit: Change in the example.

**Comment by srijan-sivakumar** [comment]
> How come we have lint failures in other files, if it is being checked in all the PRs?

@nik-redhat , currently we can ignore the lint failures. Let's finish the integration of the components and then we can take a breath and fix these linting issues.

File: parsing/redant_params_handler.py [hunk 3]
```diff
@@ -76,6 +76,76 @@ def get_client_ip_list(cls) -> list:
...
+    def get_nodes_info(cls) -> dict:
+        """
+        Returns a dictionary consisting of server info
+        Returns:
+            dict: dictionary consisting of server info
+        format of dictionary:
+        {
+            servers: [
+                        {
+                            "hostname" : server_vm1
```
Comment (srijan-sivakumar) [inline parsing/redant_params_handler.py:89]: @nishith-vihar , even the values are strings..not just the keys..

Comment (srijan-sivakumar) [reply parsing/redant_params_handler.py:89]: Anyway, I'm merging it. This change can be made in a separate patch. This fix is essential to the integration of the main, test_runner, and the runner_thread

**Review by srijan-sivakumar — APPROVED** [review]
